homeassistant/components/demo/stt.py

```diff
@@ -29,6 +32,60 @@ async def async_get_engine(
     return DemoProvider()
 
 
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Demo speech platform via config entry."""
+    async_add_entities([DemoProviderEntity()])
+
+
+class DemoProviderEntity(ProviderEntity):
+    """Demo speech API provider entity."""
+
+    @property
+    def supported_languages(self) -> list[str]:
+        """Return a list of supported languages."""
+        return SUPPORT_LANGUAGES
+
+    @property
+    def supported_formats(self) -> list[AudioFormats]:
+        """Return a list of supported formats."""
+        return [AudioFormats.WAV]
+
+    @property
+    def supported_codecs(self) -> list[AudioCodecs]:
+        """Return a list of supported codecs."""
+        return [AudioCodecs.PCM]
+
+    @property
+    def supported_bit_rates(self) -> list[AudioBitRates]:
+        """Return a list of supported bit rates."""
+        return [AudioBitRates.BITRATE_16]
```

Comment:
Not related to this PR, bit the enum values are very unclear.
Maybe we can at least improve the docstring to explain the meaning of this property?

Reply:
A suggestion is welcome. :+1: